fvm/src/gas/price_list.rs

```diff
@@ -219,7 +219,7 @@ lazy_static! {
 
         block_memory_retention: ScalingCost {
```

Comment:
Suggest a rename here to convey that this is not an additive cost, but rather a minimum baseline.